constraint/pkg/instrumentation/instrumentation.go

```diff
@@ -0,0 +1,24 @@
+// package instrumentation defines primitives to
+// support more observability throughout the contraint framework.
+package instrumentation
+
+// NamedStats covers primitive for many heterogenoues statstics and metrics
+// to be represented to consumers.
+type NamedStats struct {
+	StatsName string
+	Stats     []*Stat
+}
+
+// Stat is a Name, Value, Description tuple that may contain
+// metrics or aggregations of metrics.
+type Stat struct {
+	Name        string
```

Comment:
how is `Name` different from `StatsName`?

Reply:
see above https://github.com/open-policy-agent/frameworks/pull/283/files#r1093610950

Reply:
Unfortunately that doesn't answer the question as-written, but we can continue the discussion there.

Reply:
`NamedStats` have been removed per our offline chat.